csi/util.go

```diff
@@ -239,6 +239,26 @@ func parseJSONRecurringJobs(jsonRecurringJobs string) ([]longhornclient.Recurrin
 	return recurringJobs, nil
 }
 
+func syncMountPointDirectory(targetPath string) error {
+	d, err := os.Open(targetPath)
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+
+	if _, err := d.Readdirnames(-1); err != nil {
```

Comment:
`-1` means Readdirnames returns all file names all the names from the directory targetPath. It would be time-consuming if the targetPath directory contains a lot of file under it. What about using `d.Readdirnames(1)` or the old `os.ReadDir(targetPath)` ?

Reply:
Modified to "1", thanks